src/app/dashboard-widgets/applications-widget/applications-widget.component.spec.ts

```diff
@@ -241,17 +282,15 @@ describe('ApplicationsWidgetComponent', () => {
 
     it('should enable buttons if the user owns the space', function (this: TestingContext) {
       this.hostComponent.userOwnsSpace = true;
-      this.testedDirective.runBuildConfigs.length = 0;
-      this.testedDirective.stageBuildConfigs.length = 0;
+      this.testedDirective.buildConfigs.length = 0;
       this.detectChanges();
 
       expect(this.fixture.debugElement.query(By.css('#spacehome-applications-add-button'))).not.toBeNull();
     });
 
     it('should disable buttons if the user does not own the space', function (this: TestingContext) {
       this.hostComponent.userOwnsSpace = false;
-      this.testedDirective.runBuildConfigs.length = 0;
-      this.testedDirective.stageBuildConfigs.length = 0;
+      this.testedDirective.buildConfigs.length = 0;
       this.detectChanges();
 
       expect(this.fixture.debugElement.query(By.css('#spacehome-applications-add-button'))).toBeNull();
```

Comment:
The behavior of this component has been altered and I'm surprised there are no test additions or modifications to existing tests. Can tests be added?